src/multibody/geometry.hxx

```diff
@@ -191,18 +205,17 @@ namespace se3
     activeCollisionPairs[pairId] = false;
   }
 
-  inline CollisionResult GeometryData::computeCollision(const CollisionPair & pair) const
+  inline bool GeometryData::computeCollision(const CollisionPair & pair, fcl::CollisionResult& collisionResult) const
   {
     const Index & co1 = pair.first;     assert(co1<collisionObjects.size());
     const Index & co2 = pair.second;    assert(co2<collisionObjects.size());
 
-    fcl::CollisionRequest collisionRequest (1, false, false, 1, false, true, fcl::GST_INDEP);
-    fcl::CollisionResult collisionResult;
+    static const fcl::CollisionRequest collisionRequest (1, false, false, 1, false, true, fcl::GST_INDEP);
```

Comment:
I used `static const fcl::CollisionRequest` but I think it would be better to have a `fcl::CollisionRequest` member in `GeometryData`.